src/app/relay/mutations/CreateCommentMutation.js

```diff
@@ -101,7 +100,6 @@ class CreateCommentMutation extends Relay.Mutation {
 
     const project_media = {
       id: this.props.annotated.id,
-      log_count: this.props.annotated.log_count + 1,
```

Comment:
honestly I'm really happy to see this `+ 1` go away

Reply:
+1